sapp/ui/issues.py

```diff
@@ -488,26 +502,95 @@ def where_filter(self, filter_instance: Filter) -> "Instance":
             .where_status_is_any_of(filter_instance.statuses)
         )
 
-        if (
-            filter_instance.source_names is not None
-            and len(filter_instance.source_names) > 0
-        ):
-            builder = builder.where_source_name_is_any_of(filter_instance.source_names)
-        if (
-            filter_instance.source_kinds is not None
-            and len(filter_instance.source_kinds) > 0
-        ):
-            builder = builder.where_source_kind_is_any_of(filter_instance.source_kinds)
-        if (
-            filter_instance.sink_names is not None
-            and len(filter_instance.sink_names) > 0
-        ):
-            builder = builder.where_sink_name_is_any_of(filter_instance.sink_names)
-        if (
-            filter_instance.sink_kinds is not None
-            and len(filter_instance.sink_kinds) > 0
-        ):
-            builder = builder.where_sink_kind_is_any_of(filter_instance.sink_kinds)
+        if filter_instance.callables:
+            if type(filter_instance.callables) is not list:
+                # pyre-fixme[16]: `List` has no attribute 'get'
+                if filter_instance.callables.get("operation") == "matches":
+                    builder = builder.where_callables_matches(
+                        filter_instance.callables.get("value", [""])[0]
+                    )
+                elif len(filter_instance.callables.get("value", [])) > 0:
+                    builder = builder.where_callables_is_any_of(
+                        filter_instance.callables.get("value")
+                    )
+            else:
+                # Backward compatibility
+                # pyre-fixme[6]: Expected `List[str]` for 1st positional only
+                # parameter but got
+                # `Union[Dict[str, Union[List[str], str]], List[str]]`
```

Comment:
ditto, surprised pyre cannot infer that we have a list here. Maybe because you use `type(..) is list` instead of `isinstance()`?

Reply:
Migrated to `isinstance()`. Still pyre fails!